.gitignore

```diff
@@ -37,3 +37,6 @@ yarn-error.log*
 *.tsbuildinfo
 
 .idea
+
+# Sentry Auth Token
+.sentryclirc
```

Comment:
can't the auth token be placed in .env files? also, if we can't use .env, how to supply this auth token to vercel?

Reply:
This could be done using .env variables, this was automatically set up by the sentry default installation tool. https://docs.sentry.io/product/cli/configuration/#configuration-values